### ISSUE TITLE
learning_rate  issue?

### ISSUE THREAD

**Issue by bobo0810** [eriklindernoren/PyTorch-YOLOv3#70]
Hi , didn't the learning rate update during the training phase?

**Comment by XCRobert** [comment]
Yes, `train.py`uses Adam as the optimizer
